From 1883d299bf853dac75cd67c012b9f074b0a4dbef Mon Sep 17 00:00:00 2001
From: nicm <nicm>
Date: Mon, 29 May 2017 07:58:33 +0000
Subject: [PATCH] Do not factor in screen_hsize() for the visible copy mode
 screen when adjusting the selection, it should never have any useful history
 (and when it does, after resize, we shouldn't use it). From Michal Mazurek.

---
 window-copy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/window-copy.c b/window-copy.c
index 6d1a9f..c8807c 100644
--- a/window-copy.c
+++ b/window-copy.c
@@ -1458,7 +1458,7 @@ window_copy_adjust_selection(struct window_pane *wp, u_int *selx, u_int *sely)
 	}
 
 	*selx = sx;
-	*sely = screen_hsize(s) + sy;
+	*sely = sy;
 	return (relpos);
 }
 
-- 
2.13.5

